src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -81,6 +83,17 @@ public class AzureServiceBusMessagePumpSettings
         /// <returns>Connection string to authenticate with</returns>
         public async Task<string> GetConnectionStringAsync()
         {
+            ILogger logger = 
+                _serviceProvider.GetService<ILogger<AzureServiceBusMessagePumpSettings>>() 
+                ?? NullLogger<AzureServiceBusMessagePumpSettings>.Instance;
+
+            logger.LogSecurityEvent("Get Azure Service Bus connection string", new Dictionary<string, object>
```

Comment:
Please refresh my memory; will it track security events out-of-the-box or is this an opt-in?

Reply:
Hmm, not, the option to opt-out for security events is something we added to the secret store functionality in the security repo. We didn't had security events logging in the messaging repo, and therefore no opt-out functionality.

Reply:
I would add it then because by default I would prefer not to have these since they can pollute our telemetry

Reply:
OK, done. Added options to opt-in for security events.